test/TestMain.hx

```diff
@@ -1,5 +1,3 @@
-package ;
```

Comment:
I guess it would need to cover both cases.. Either requiring empty packages or complaining about them when they're disallowed. `EmptyPackageCheck`.